### ISSUE TITLE
Move Environment Configuration from Popup to Tab

### ISSUE THREAD

**Issue by samuelwford** [usebruno/bruno#2363]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Please consider changing the environment configuration popup window to a tab, same as how collection settings works.

### Mockups or Images of the feature

Instead of using this presentation style for environments:
<img width="1112" alt="Screenshot 2024-05-24 at 11 36 55 AM" src="https://github.com/usebruno/bruno/assets/569369/5c605f24-5f3a-47de-9bfb-868bd7776f97">

Consider using this presentation style:
<img width="1112" alt="Screenshot 2024-05-24 at 11 36 49 AM" src="https://github.com/usebruno/bruno/assets/569369/43785e46-ae58-4bb9-961e-1819028897e3">

**Comment by bitfactory-edo-freriks** [comment]
Yes please, this would be a great addition, the scrolling inside the modal is also a bit clunky, and when you add a new variable after the list is already long enough, it doesn't automatically scroll down.

**Comment by anusree-bruno** [comment]
This is done. Please checkout latest version.
